File: 1468-check-if-n-and-its-double-exist/1468-check-if-n-and-its-double-exist.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        n = len(arr)
        dict = {}
        for i in range(n):
            comp, res = arr[i] * 2, arr[i] / 2
            if comp in dict or res in dict:
                return True
            else:
                dict[arr[i]] = dict.get(arr[i], 0) + 1
        return False

File: 1468-check-if-n-and-its-double-exist/1468-check-if-n-and-its-double-exist.cs
public class Solution {
    public bool CheckIfExist(int[] arr) {
        int i, j;
        for (i = 0; i < arr.Length; i++) {
            for (j = i + 1; j < arr.Length; j++) {
                if (arr[i] == 2 * arr[j] ||
                    (arr[j] % 2 == 0 && arr[i] == arr[j] / 2))
                    return true;
            }
        }
        return false;
    }
}